docker/README.md

```diff
@@ -327,10 +329,40 @@ Instructions on how to build standalone docker images (for development or
 production), outside of `docker-compose`, see the
 [docker docs](../docs/DOCKER.md).
 
+### Using bitcoind version
+If you are using the bitcoind version of the compose file i.e `docker-compose-bitcoind.yml`, follow these additional instructions:
+
+#### Start Bitcoin Node with bitcoind using Docker Compose
+To launch the Bitcoin node using bitcoind in the regtest network using Docker Compose, use the following command:
+```shell
+$  NETWORK="regtest" docker-compose -f docker-compose-bitcoind.yml up
+```
+
+#### Generating RPCAUTH
+Since using `rpcuser` and `rpcpassword` is deprecated in bitcoind core we need to use `rpcauth` instead. You could generate `rpcauth` using [rpcauth.py script](https://github.com/bitcoin/bitcoin/blob/master/share/rpcauth/rpcauth.py) from bitcoin core.
+
+#### Mining in regtest using bitcoin-cli
+For mining in the regtest network, you can use `bitcoin-cli` with the following command:
+```shell
+$  bitcoin-cli -chain=regtest -rpcauth=<your_rpc_auth> generatetoaddress 10 <your_regtest_address>
+```
+
+#### Generating Wallet in regtest using Bitcoin Core
+To create a wallet in the regtest network using `bitcoin-cli`, use the following command:
+```shell
+$  bitcoin-cli -chain=regtest -rpcauth=<your_rpc_auth> createwallet <wallet_name>
```

Comment:
If we do the above, then we don't need a wallet in `bitcoind` and can only use the one in `lnd`. Probably easier for the users to understand.